book/src/styles/custom.css

```
@@ -0,0 +1,19 @@
+/* logfwd docs — brand overrides */
+
+:root {
+  /* Accent — a clean teal that reads well on both light and dark */
+  --sl-color-accent-low: #1a3a4a;
+  --sl-color-accent: #2d8ca8;
+  --sl-color-accent-high: #b8e4f0;
+
+  /* Typography — system font stack for speed */
+  --sl-font: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Stylelint (17.6.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)